components/DisplayOwnCollection/index.js

```diff
@@ -0,0 +1,36 @@
+import { useState } from "react";
+
+const ownCollection = [
+  /* {
+    id: "1",
+    name: "Name des Comics",
+  },
+  {
+    id: "2",
+    name: "Name des Comics",
+  },
+  {
+    id: "3",
+    name: "Name des Comics",
+  }, */
```

Comment:
why is this in comments?

Reply:
this was a test to see, if the data is displayed, when it exists and when the "empty collection" message is displayed, when the collection is empty.